nemoguardrails/colang/v2_x/library/llm.co

```diff
@@ -210,6 +213,8 @@ flow llm generate interaction continuation flow -> $flow_name
   log "generated flow continuation: `{$flow_info}`"
   $exists = await CheckValidFlowExistsAction(flow_id=$flow_info.name)
 
+  await RetrieveRelevantChunksAction()
```

Comment:
And I think this one needs to be moved above the `await GenerateFlowContinuationAction`